src/getMigrateDashboards.ts

```diff
@@ -41,7 +42,7 @@ export const getMigrateDashboards =
     const dashboardsStructure =
       contentServer.children?.ui.children?.dashboards.children?.structure!;
 
-    const mapOfFolderIds = _getMapOfFolderIds(dashboardsStructure);
+    const mapOfFolderIds = _getFilesAncestry(dashboardsStructure);
```

Comment:
The name is not appropriate anymore: it should be `filesAncestry` or `mapOfAncestors`